src/app/components/Layout/Header/MenuUser/index.tsx

```diff
@@ -36,7 +35,15 @@ const MenuUser: React.FunctionComponent = () => {
       color="inherit"
     >
       <Icon />
-      <TextLogin>{user ? `${JSON.stringify(user)} (logOut)` : "login"}</TextLogin>
+      <TextLogin>
+        {user ? (
+          <UserName>
+            {JSON.stringify(user).slice(1, -1)} &nbsp;<span>log out</span>
```

Comment:
check trim() (if you wanted to remove whitespaces)

Reply:
Here I am removing the quotes